src/test/java/com/example/demo/model/BookTest.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BookTest {
+
+    @Test
+    public void setAuthorTest() {
+        Book book = new Book();
```

Comment:
i tu mozna ladnie to rozbic
// given
Book book = new Book();

// when
book.setAuthor("James Williams");

// then
assertEquals("James Williams", book.getAuthor());